avaframe/in1Data/getInput.py

```diff
@@ -497,3 +504,169 @@ def fetchReleaseFile(inputSimFiles, releaseScenario, cfgSim, releaseList):
             cfgSim['INPUT'].pop(scenario + '_' + 'relThCi95')
 
     return releaseScenarioPath, cfgSim
+
+
+def createReleaseStats(avaDir, cfg):
+    """ create a csv file with info on release shp file on:
+        max, mean and min elevation, slope and projected and real area
+
+        Parameters
+        -------------
+
+        Returns
+        --------
+        fPath: pathlib Path
+            file path
+    """
+
+    # fetch input data (dem and release area shp file)
+    inputData = getInputDataCom1DFA(avaDir)
+
+    # get line from release area polygon
+    relFiles = inputData['relFiles']
+
+    # initialize dem and get real area of dem
+    dem = IOf.readRaster(inputData['demFile'], noDataToNan=True)
+    dem['originalHeader'] = dem['header'].copy()
+    methodMeshNormal = cfg.getfloat('GENERAL', 'methodMeshNormal')
+    # get normal vector of the grid mesh
+    dem = DFAtls.getNormalMesh(dem, methodMeshNormal)
+    dem = DFAtls.getAreaMesh(dem, methodMeshNormal)
+    header = dem['header']
+    csz = header['cellsize']
+
+    # define parameters to be saved
+    relFeatures = []
+    relMaxZ = []
+    relMinZ = []
+    relSlope = []
+    relArea = []
+    relAreaP = []
```

Comment:
To make this more readable, I suggest switching to a dict with keys here, especially in connection with my comment below